package.json

```diff
@@ -119,5 +91,21 @@
     "prepack": "pinst --disable",
     "postpack": "pinst --enable"
   },
-  "packageManager": "yarn@3.1.1"
+  "packageManager": "yarn@3.1.1",
+  "resolutions": {
```

Comment:
Do we need resolutions, is this for the new linking? Asking because I can see it's referring to you local space or it's just a example how it must look like